client/jetpack-onboarding/main.jsx

```diff
@@ -135,12 +136,14 @@ class JetpackOnboardingMain extends React.PureComponent {
 			steps,
 			translate,
 		} = this.props;
+		const basePath = '/jetpack/start';
+		const pageTitle = get( STEP_TITLES, stepName ) + ' ‹ ' + translate( 'Jetpack Start' );
 
 		return (
 			<Main className="jetpack-onboarding">
-				<DocumentHead
-					title={ get( STEP_TITLES, stepName ) + ' ‹ ' + translate( 'Jetpack Start' ) }
-				/>
+				<DocumentHead title={ pageTitle } />
+				<PageViewTracker path={ [ basePath, stepName, ':site' ].join( '/' ) } title={ pageTitle } />
```

Comment:
I think we don't want to `translate` anything we feed to analytics: IIUC, we try to avoid having data in different languages for essentially the same event, which would make querying/funneling so much harder.

I think this means we'll have to continue to duplicate `pageTitle` -- a translated version for `<DocumentHead />`, and an untranslated one here.

Reply:
Ah, good find. Addressed in fe4bd1209dc104f847d382394a6f036be333924d. Shame we had to duplicate those, but 🤷‍♂️